src/CMakeLists.txt

```diff
@@ -20,13 +20,41 @@ include(../cmake/limit_jobs.cmake)
 
 set (CONFIG_VERSION "${CMAKE_CURRENT_BINARY_DIR}/Common/config_version.h")
 set (CONFIG_COMMON "${CMAKE_CURRENT_BINARY_DIR}/Common/config.h")
+set (GIT_INFO "${CMAKE_CURRENT_BINARY_DIR}/Common/git_info.h")
 
 include (../cmake/version.cmake)
 message (STATUS "Will build ${VERSION_FULL} revision ${VERSION_REVISION} ${VERSION_OFFICIAL}")
 configure_file (Common/config.h.in ${CONFIG_COMMON})
 configure_file (Common/config_version.h.in ${CONFIG_VERSION})
 configure_file (Core/config_core.h.in "${CMAKE_CURRENT_BINARY_DIR}/Core/include/config_core.h")
 
+set(GIT_EXECUTABLE /usr/bin/git)
```

Comment:
Probably we need a case when git is unavailable or we build not from repo.

Reply:
ok.